src/widgets/WidgetMainQmc.vue

```diff
@@ -107,6 +116,95 @@ export default defineComponent({
       return this.cards.slice(0, 4)
     },
   },
+  created() {
+    this.getPrimaryProduct()
+    this.getCardAccountFee()
+    this.getCardStandardInterestRate()
+    this.createTitle()
+    this.createSubtitle()
+  },
+  methods: {
+    getPrimaryProduct() {
+      this.primaryProduct.productType = this.products[0].productType
+      this.primaryProduct.productItems = this.products[0].productItems
+    },
+    getCardAccountFee() {
+      this.cardAccountFee = this.cards.find(
+        card => card.id === 'QMC1'
+      )?.annualFee
```

Comment:
Just confirming: does only qmc1 have a card account fee?

Reply:
qmc1 is the only account fee valid for `WidgetMainQmc.vue`